test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1599,9 +1600,9 @@ def forward(self, x: int, y: int):
         self.run_test(ArithmeticModule(), (x, y))
 
     # Outputs that are always None are removed.
-    # We don't know Optional.elem_type, so we can't construct a valid Optional.
-    # Tests for Optional outputs (control flow with None in one branch,
-    # not-None in another) are in test_pytorch_onnx_no_runtime.py.
+    # Issue 84130: ONNX ignores mustNone() node, while pytorch
```

Comment:
track https://github.com/pytorch/pytorch/issues/84130.
This is a workaround here, but if any reviewer thinks this is a bad idea even for a temporary workaround, I understand.

Reply:
cc @BowenBao What do you think? Can we merge this?

Reply:
sgtm

Reply:
Thanks! I will work on this right away.